design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,147 @@
+type Mutation {
+    requestNegotiableQuote(
+        cart_id: ID! @doc(description: "Cart ID")
+        quote_name: String @doc(description: "Quote name")
+        comment: String @doc(description: "Comment")
+        files: [FileInput] @doc(description: "Attached files")
```

Comment:
Think `[FileInput]` should probably be `[FileInput!]`, right? Can't think of a use for a `null` in `files`